extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -397,6 +419,19 @@ export class AmpStoryPage extends AMP.BaseElement {
     return this.isPrerenderActivePage_;
   }
 
+  /**
+   * Returns true if the page is the last amp-story-page in the amp-story.
+   * @return {boolean}
+   */
+  isLastPage() {
+    if (this.isLastPage_ != null) {
+      return this.isLastPage_;
+    }
+    const selector = 'amp-story-page:last-of-type';
+    const selectorNode = this.win.document.querySelector(selector);
+    return selectorNode == this.element;
```

Comment:
Maybe we can make this a tiny bit easier by using the `matches` helper:

https://github.com/ampproject/amphtml/blob/ea2e7f696feca04e4fc0be41f4dc60c9a240904d/src/dom.js#L483-L500

Reply:
Done. Love it, this is great!